sphinx/writers/latex.py

```diff
@@ -572,6 +572,22 @@ def __init__(self, document, builder):
         else:
             self.elements['date'] = format_date(builder.config.today_fmt or _('%b %d, %Y'),  # type: ignore  # NOQA
                                                 language=builder.config.language)
+
+        self.numfig_secnum_depth = -1
```

Comment:
I feel current `__init__()` is too fat. So I propose you to separate this calculation to independent method or function.